### DISCUSSION TITLE
Easy Model Initialization and Switching

### DISCUSSION THREAD

**Discussion by Okeke-Stephen** [imartinez/privateGPT#1494] [Ideas]
Thank you for this great initiative and contributions. 
It is a little bit tricky to initiate and use new models. I think it will be better to simply define and add our own model. For instance, stuff like:

llm = (----, ---, ---,)

Embedding = (---, ---, ---)

Then, quickly calling and using the models would be better.

This can also apply to other open-source models, wherein we define the models and easily fix them into the rest of the code.
Thanks once more.

**Comment by tedraymond** [comment]
Mode= []

Can be tuple,

Then add additional case statements.

This will give users the option of additional model, even remote api llm andpoints
